modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/ImportStyleBookDisplayContext.java

```diff
@@ -23,48 +23,46 @@ public ImportStyleBookDisplayContext(RenderRequest renderRequest) {
 		_renderRequest = renderRequest;
 	}
 
-	public List<String> getStyleBookEntryZipProcessorImportResultEntryNames(
-		StyleBookEntryZipProcessorImportResultEntry.Status status) {
+	public List<String> getStyleBookImporterResultEntryNames(
+		StyleBookEntryImporterImportResultEntry.Status status) {
 
-		List<StyleBookEntryZipProcessorImportResultEntry>
-			styleBookEntryZipProcessorImportResultEntries =
-				_getStyleBookEntryZipProcessorImportResultEntryNames();
+		List<StyleBookEntryImporterImportResultEntry>
+			styleBookImporterResultEntries =
+				_getStyleBookImporterResultEntryNames();
 
-		if (ListUtil.isEmpty(styleBookEntryZipProcessorImportResultEntries)) {
+		if (ListUtil.isEmpty(styleBookImporterResultEntries)) {
 			return null;
 		}
 
 		return TransformUtil.transform(
-			styleBookEntryZipProcessorImportResultEntries,
-			styleBookEntryZipProcessorImportResultEntry -> {
-				if (styleBookEntryZipProcessorImportResultEntry.getStatus() !=
+			styleBookImporterResultEntries,
+			styleBookEntryImporterImportResultEntry -> {
+				if (styleBookEntryImporterImportResultEntry.getStatus() !=
 						status) {
 
 					return null;
 				}
 
-				return styleBookEntryZipProcessorImportResultEntry.getName();
+				return styleBookEntryImporterImportResultEntry.getName();
 			});
 	}
 
-	private List<StyleBookEntryZipProcessorImportResultEntry>
-		_getStyleBookEntryZipProcessorImportResultEntryNames() {
+	private List<StyleBookEntryImporterImportResultEntry>
+		_getStyleBookImporterResultEntryNames() {
 
-		if (_styleBookEntryZipProcessorImportResultEntries != null) {
-			return _styleBookEntryZipProcessorImportResultEntries;
+		if (_styleBookImporterResultEntries != null) {
+			return _styleBookImporterResultEntries;
 		}
 
-		_styleBookEntryZipProcessorImportResultEntries =
-			(List<StyleBookEntryZipProcessorImportResultEntry>)
-				SessionMessages.get(
-					_renderRequest,
-					"styleBookEntryZipProcessorImportResultEntries");
+		_styleBookImporterResultEntries =
+			(List<StyleBookEntryImporterImportResultEntry>)SessionMessages.get(
+				_renderRequest, "styleBookImporterResultEntries");
 
-		return _styleBookEntryZipProcessorImportResultEntries;
+		return _styleBookImporterResultEntries;
 	}
 
 	private final RenderRequest _renderRequest;
-	private List<StyleBookEntryZipProcessorImportResultEntry>
-		_styleBookEntryZipProcessorImportResultEntries;
+	private List<StyleBookEntryImporterImportResultEntry>
+		_styleBookImporterResultEntries;
```

Comment:
Once again the variable name must match the entity type: `_styleBookEntryImporterImportResultEntries`